src/app/wesprzyj/platnosc/_components/StripeElementsWrapper.tsx

```diff
@@ -0,0 +1,78 @@
+"use client";
+
+import { Elements } from "@stripe/react-stripe-js";
+import { loadStripe, type StripeElementsOptions } from "@stripe/stripe-js";
+
+const stripePublishableKey = process.env.NEXT_PUBLIC_STRIPE_PUBLISHABLE_KEY;
+
+if (!stripePublishableKey) {
+  throw new Error("Missing NEXT_PUBLIC_STRIPE_PUBLISHABLE_KEY env variable");
+}
+
+const stripePromise = loadStripe(stripePublishableKey);
```

Comment:
Maybe we could encapsulate this and the above `env` check within a function? No strong opinions about this TBH